src/main.cpp

```diff
@@ -16,12 +17,12 @@
 int main(int argc, char** argv)
 {
     int init_frame_id = 0;
-
+    ConfigReader reader("F:\\repos\\svo\\config.yaml");
```

Comment:
use relative paths: "./config.yaml"

Reply:
It doest work if i use relative path to config((